lib/java/com/google/android/material/lists/viewHolders/MaterialViewHolder.java

```diff
@@ -0,0 +1,63 @@
+package com.google.android.material.lists.viewHolders;
+
+import android.content.res.Resources;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import androidx.annotation.DimenRes;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.google.android.material.R;
+
+/** A standard view holder that all specific Material view holders extend */
+public abstract class MaterialViewHolder extends RecyclerView.ViewHolder {
+
+  protected MaterialViewHolder(@NonNull View itemView) {
+      super(itemView);
+  }
+
+  protected MaterialViewHolder(int layout, @NonNull ViewGroup parent, @DimenRes int insetDividerXRes) {
```

Comment:
```suggestion
  protected MaterialViewHolder(@LayoutRes int layout, @NonNull ViewGroup parent, @DimenRes int insetDividerXRes) {
```

Reply:
nit: Layout resource annotation to guide a layout resource int to be passed into MaterialViewHolder's `layout` parameter.

Good catch